### ISSUE TITLE
No longer Compatible 

### ISSUE THREAD

**Issue by xInsanityx1022** [Platonymous/Stardew-Valley-Mods#122]
![image](https://user-images.githubusercontent.com/86631130/144548045-1a71b7f8-7984-46c1-b94f-43bb0f6837f5.png)

PyTK is no longer compatible with the smapi update :/ i can't run my mods anymore. please update when you can.

Thank you

**Comment by Entgan** [comment]
Yep, seems some steam users like myself are in this situation. Hate to rush a developer but I basically all of my mods I use use this in some form or another so I can't even play without it being basically vanilla.
